docs/runtime_options.md

```diff
@@ -125,8 +125,7 @@ cAdvisor stores the latest historical data in memory. How long of a history it s
 --application_metrics_count_limit=100: Max number of application metrics to store (per container) (default 100)
 --collector_cert="": Collector's certificate, exposed to endpoints for certificate based authentication.
 --collector_key="": Key for the collector's certificate
---disable_metrics=tcp,advtcp,udp,sched,process,hugetlb: comma-separated list of metrics to be disabled. Options are 'disk', 'network', 'tcp', 'advtcp', 'udp', 'sched', 'process', 'hugetlb', 'cpuset'. Note: tcp and udp are disabled by default due to high CPU usage. (default tcp,advtcp,udp,sched,process,hugetlb,cpuset)
---disable_specific_metrics="": Comma-separated list of metrics not to be enabled. This list comprises of exact metric names and/or regex patterns.It differs from the 'disable_metrics' in that 'disable_specific_metrics' filters specific metrics, while 'disable_metrics' filters a certain type of metrics.Besides, 'disable_specific_metrics' will only disable metrics being exported by Prometheus.(default "")
+--disable_metrics=process,tcp,udp,sched: comma-separated list of metrics to be disabled. Options are 'disk', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process'. Note: tcp and udp are disabled by default due to high CPU usage. (default process,tcp,udp,sched)
```

Comment:
List is incomplete. See:
* [`ignoreMetrics`](https://github.com/google/cadvisor/blob/bfe982b346633ca5e87021019e9e2ab75aa0d7fe/cmd/cadvisor.go#L83) for default ignore list
* [`ignoreWhitelist`](https://github.com/google/cadvisor/blob/bfe982b346633ca5e87021019e9e2ab75aa0d7fe/cmd/cadvisor.go#L98) for all possible values.

Reply:
Updated the text from the out from the latest release:

```
docker run --rm -ti gcr.io/cadvisor/cadvisor:latest -version
cAdvisor version v0.38.6 (30515557)
```

```
❯ docker run --rm -ti gcr.io/cadvisor/cadvisor:latest --help
Usage of /usr/bin/cadvisor:
	
  <snip>

  -disable_metrics metrics
    	comma-separated list of metrics to be disabled. Options are 'accelerator', 'cpu_topology','disk', 'diskIO', 'memory_numa', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb', 'referenced_memory', 'resctrl'. (default referenced_memory,cpu_topology,resctrl,udp,advtcp,sched,hugetlb,memory_numa,tcp,process)

  <snip>
```

Reply:
Added `cpuset` from 0.3.9.

https://github.com/google/cadvisor/blob/release-v0.39/cmd/cadvisor.go#L146